templates/common/infra/bicep/core/host/appservice.bicep

```diff
@@ -0,0 +1,100 @@
+param environmentName string
```

Comment:
Thoughts on having the `appService` module also be the entry point that creates the underlying ASP? If the ASP already exists use it, otherwise create it?

Would make it easier for folks to know that they just need an `app` and not have to work about including the correct set of dependent resources to make it happen.

Then at some point we can could consider taking it a step further where there is just an `app` module and based on some settings it sometimes creates an app serivce, container app, function app, etc.

Reply:
Yes, we should consider that for future improvement.  There's some gotchas with that that I would like to test and explore in-depth as a separate task.